tests/Session/test_commands.py

```diff
@@ -0,0 +1,21 @@
+from src.Session.commands import Commands
+from unittest.mock import patch
```

Comment:
What does `patch` do and how is it being implemented in this test?

Reply:
It's used on line 13. I wanted a way to test when we request user input and found this way to do it. So I think how it works is it mocks the method you specify (in this case it's `input()` in `commands.py`) and then on line 16, we pass in what values we want as user input. 

So for example in `create_profile()`, we have this code:
```
# CREATE_PROFILE_NAME = "What would you like to name your profile?: "
profile_name = input(CREATE_PROFILE_NAME)
```

When the user is prompted with the question, our test passes in `"testProfile"` as a response, so `profile_name` will be set to `"testProfile"`

Reply:
I see, thank you very much for explaining!